docs/source/deploying.rst

```diff
@@ -84,59 +84,113 @@ The following resources explain how to set up Dask on a variety of local and dis
 
 Cloud
 -----
-|Coiled|_ **is recommended for deploying Dask on the cloud.**
-Though there are other options you may consider depending on your specific needs:
 
-- `Coiled <https://coiled.io?utm_source=dask-docs&utm_medium=deploying>`_: Commercial Dask deployment option, which handles the creation and management of Dask clusters on cloud computing environments (AWS, GCP, and Azure).
-- `Dask Cloud Provider <https://cloudprovider.dask.org/en/latest/>`_: Constructing and managing ephemeral Dask clusters on AWS, DigitalOcean, Google Cloud, Azure, and Hetzner.
-- `Dask-Yarn <https://yarn.dask.org>`_: Deploy Dask on YARN clusters, such as are found in traditional Hadoop installations.
+Deploying on commercial cloud like AWS, GCP, or Azure is convenient because you can quickly scale out to many machines for just a few minutes, but also challenging because you need to navigate awkward cloud APIs, manage remote software environments with Docker, send data access credentials, make sure that costly resources are cleaned up, etc..  The following solutions help with this process.
```

Comment:
I don't think we need the extra `.` in this case

```suggestion
Deploying on commercial cloud like AWS, GCP, or Azure is convenient because you can quickly scale out to many machines for just a few minutes, but also challenging because you need to navigate awkward cloud APIs, manage remote software environments with Docker, send data access credentials, make sure that costly resources are cleaned up, etc.  The following solutions help with this process.
```